tests/test_interface_private_vlan.rb

```diff
@@ -55,6 +55,12 @@ def interface_cleanup_pvlan
   end
 
   def test_pvlan_mapping
+    if validate_property_excluded?('interface', 'feature_vlan')
```

Comment:
``Interface.new('vlan13')`` attempts to execute unsupported yaml, so I added this bail-out